classes/pdf/HTMLTemplateOrderSlip.php

```diff
@@ -198,9 +198,17 @@ public function getProductTaxesBreakdown()
 				foreach ($tax_amount as $tax_id => $amount)
 				{
 					$tax = new Tax((int)$tax_id);
-					$tax_rate = $tax->rate;
```

Comment:
This is will cause a PHP notice because we use this variable [here](https://github.com/PrestaShop/PrestaShop/pull/2087/files#diff-c8f6ff671680366c26adcbe57c61748cR213)